### DISCUSSION TITLE
When I Click One Button Another Time I CLick Not Works

### DISCUSSION THREAD

**Discussion by Ignelis91** [LunarTaku/djs-ticket-system#5] [Error]
Event: https://sourceb.in/8tPV9zSXP3
command: https://sourceb.in/TlcYBspcj2

**Comment by LunarTaku** [comment]
try placing `once: false` below the `name:`.

**Reply by Ignelis91** [reply]
Still Same

**Reply by LunarTaku** [reply]
can you tell me what button is creating this error? And is it logging an error in the console?

**Reply by Ignelis91** [reply]
Its Not Logining Error Buttons I Click Whatever Button Then Next Button is gona not work or same button or other wil not work

**Reply by LunarTaku** [reply]
alright one second
=

**Reply by Ignelis91** [reply]
Ok

**Reply by LunarTaku** [reply]
i have no idea why this is happening, this has never happend to me. I tested this system on 4 servers and over 200 people and it has never happend, try asking in general supoort they may help you.

**Reply by Ignelis91** [reply]
idk it can be bc i dont have button handler

**Comment by LunarTaku** [comment]
@Ignelis91 Please check the files now i may have solved your issue!

**Reply by Ignelis91** [reply]
Still Same

**Reply by LunarTaku** [reply]
oh damn
